Sources/MeiliSearch/QueryParameters/DocumentsQuery.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+public class DocumentsQuery: Queryable {
+  private var limit: Int?
+  private var offset: Int?
+  private var fields: [String]
+
+  init(limit: Int? = nil, offset: Int? = nil, fields: [String]? = nil) {
```

Comment:
Actually, there is a subtle difference between those implementations. Since the engine supports the `/documents?fields=` (without no value), which response with empty objects, I opted to receive as a default value `nil`, so if somebody wants that response they can do it :)